src/include/common/statement.h

```diff
@@ -68,22 +69,22 @@ class Statement {
 
  private:
   // logical name of statement
-  std::string statement_name;
+  std::string statement_name_;
 
   // query string
-  std::string query_string;
+  std::string query_string_;
 
   // first token in query
-  std::string query_type;
+  std::string query_type_;
```

Comment:
Since it will take me a little longer (perhaps after the new year) to get that patch in, we will have to live with this hack till then?